packages/api-reference/src/components/Util/Style.vue

```diff
@@ -3,6 +3,8 @@
 
   We have to use this to inject stylesheets dynamically into the
   page right now because we can't inject into <head> in SSR
+
+  Warn: this does NOT work on SSR, we shouldn't use this component
```

Comment:
We can probably remove the comment above that says this is to support SSR since apparently it doesn't work 😅 

I don't really have a strong opinion about how we want to do style injection and support SSR 🤷‍♀️